appendices.tex

```diff
@@ -202,41 +203,105 @@ \subsubsection{Migrating your ledger}
+    \item Create a new database. 
+        The database name is usually very similar to the company name. 
+        Do not create the database using \texttt{setup.pl} as the restore in the next step will setup the company.   The \texttt{psql} syntax is:
+        
+        \texttt{\$ psql -h [database host] -U [database admin user] -c 'CREATE DATABASE [new company name]'}
+        
+        For example:
+        
+        \texttt{\$ psql -h localhost -U lsmb\_dbadmin -c 'CREATE DATABASE newcompany'}
+    \item Restore the "data" backup. The \texttt{psql} syntax is:
+    
+        \texttt{\$ psql -h [database host] -U [database admin user] [new company name] < [company backup file]}
+        
+        For example:
+        
+        \texttt{\$ psql -h localhost -U lsmb\_dbadmin newcompany < lsmb-db.sqlc}
+    \item Read the beginning section of the PostgreSQL release notes for each PostgreSQL version from the old to the new and determine
+        if there are any additional steps that must be performed on the database after the restore. 
+        If there are, do them now.
+    \item Verify all data restored and that LedgerSMB is working correctly.
+    \item Replace the old LSMB installation with the new installation. 
+        The exact steps to accomplish this varies based on how LSMB was originally installed.
+    \item Allow users to use LSMB.
+\end{enumerate}
+
+\subsection{Notes on migration from PostgreSQL 8.3 (or earlier) to 8.4 - deprecated }
 \label{subsec-migration-pre84-to84plus}
 
-% Performance benefit due to built in support for recursive queries only available
-% after the next setup.pl run
-% Also, after next setup.pl run, one should uninstall the tablefunc extension
+\vspace{5mm}
+\begin{center}
+\large{The content of this sub-section is deprecated, unedited, and only remains for historical reference purposes.}
+\end{center}
+\vspace{5mm}
+
+Performance benefit due to built in support for recursive queries only available
+after the next setup.pl run
+Also, after next setup.pl run, one should uninstall the tablefunc extension
+
+In 9.1+ may need to \texttt{DROP EXTENSION tablefunc;}.
 
-% 9.1+ DROP EXTENSION tablefunc;
-% 8.4, 9.0: use uninstall_tablefunc.sql from the contrib directory
+In 8.4, to 9.0 may need to  \texttt{uninstall\_tablefunc.sql} from the PostgreSQL contrib directory.
 
-LedgerSMB 1.3 uses some extension modules for versions 8.3 and before for functionality
-that has been built into 8.4 and later. To make use of the (faster) built in version
+LedgerSMB 1.3 uses some extension modules for versions PostgreSQL 8.3 and before for functionality
+that has been built into PostgreSQL 8.4 and later. To make use of the (faster) built in version
 of that functionality, the following restore procedure should be used.
 
 \begin{enumerate}
-\item Migrate the database to the new function as described in section @@@ TODO
+\item Migrate the database to the new function as described in section

```

Comment:
Which section? (The @@@ TODO was meant to add a reference to the section (by bookmark))

Reply:
Yes it was.  My first thought was to delete this section completely.  But after I looked at it, it seemed like there was some historical information that should stick around.  There would actually be a lot of work to try to figure out what exactly was the intent.  So I just tried to remove the obvious. I don't think this section has any integrity.  

Hence the comment at the beginning, "The content of this sub-section is deprecated, unedited, and only remains for historical reference purposes".

I am going to delete " as described in section" and just leave it, unless someone else who was involved at the time cares to sort it out.